new-integration-tests/src/test/java/oracle/weblogic/kubernetes/assertions/TestAssertions.java

```diff
@@ -28,6 +28,26 @@
     return Operator.isRunning(namespace);
   }
 
+  /**
+   * Check if Traefik is running.
+   *
+   * @param namespace in which the traefik is running
+   * @return true if running false otherwise
+   */
+  public static Callable<Boolean> isTraefikRunning(String namespace) {
+    return Traefik.isRunning(namespace);
+  }
+
+  /**
+   * Check if Traefik is ready.
+   *
+   * @param namespace in which the traefik is running
```

Comment:
namespace in which to check if there are Traefik pods in ready state

Reply:
fixed